reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Csv.md

```diff
@@ -18,113 +18,123 @@ Converts object properties in comma-separated value (CSV) format into CSV versio
+$J = Start-Job -ScriptBlock { Get-Process } | ConvertTo-Csv
+$Header = "State", "MoreData", "StatusMessage", "Location", "Command", "StateInfo", "Finished", "InstanceId", "Id", "Name", "ChildJobs", "PSBeginTime", "PSEndTime", "PSJobTypeName", "Output", "Error", "Progress", "Verbose", "Debug", "Warning", "Information"
 # Delete header from $J
-PS C:\> $J = $J[0], $J[2..($J.count - 1)]
-PS C:\> $J | ConvertFrom-Csv -Header $Header
+$J = $J[1..($J.count - 1)]
+$J | ConvertFrom-Csv -Header $Header
+```
+
+```Output
+State         : Running
 MoreData      : True
 StatusMessage :
 Location      : localhost
-Command       : get-process
-State         : Running
+Command       : Get-Process
+StateInfo     : Running
 Finished      : System.Threading.ManualResetEvent
 InstanceId    : 6fcb6578-7f42-4d93-9f23-9937f6aac1a2
-SessionId     : 1
-Name          : Job1
+Id            : 83
+Name          : Job83
 ChildJobs     : System.Collections.Generic.List`1[System.Management.Automation.Job]
+PSBeginTime   : 12/17/2018 16:15:52
+PSEndTime     :
+PSJobTypeName : BackgroundJob
 Output        : System.Management.Automation.PSDataCollection`1[System.Management.Automation.PSObject]
 Error         : System.Management.Automation.PSDataCollection`1[System.Management.Automation.ErrorRecord]
 Progress      : System.Management.Automation.PSDataCollection`1[System.Management.Automation.ProgressRecord]
-Verbose       : System.Management.Automation.PSDataCollection`1[System.String]
-Debug         : System.Management.Automation.PSDataCollection`1[System.String]
-Warning       : System.Management.Automation.PSDataCollection`1[System.String]
-StateChanged  :
+Verbose       : System.Management.Automation.PSDataCollection`1[System.Management.Automation.VerboseRecord]
+Debug         : System.Management.Automation.PSDataCollection`1[System.Management.Automation.DebugRecord]
+Warning       : System.Management.Automation.PSDataCollection`1[System.Management.Automation.WarningRecord]
+Information   : System.Management.Automation.PSDataCollection`1[System.Management.Automation.InformationRecord]
 ```
 
-This example shows how to use the Header parameter of **ConvertFrom-Csv** to change the names of properties in the resulting imported object.
+This example shows how to use the **Header** parameter of `ConvertFrom-Csv` to change the names of properties in the resulting imported object.
 
-The first command uses the **Start-Job** cmdlet to start a background job that runs a **Get-Process** command on the local computer.
-A pipeline operator (|) sends the resulting job object to the **ConvertTo-CSV** cmdlet, which converts the job object to CSV format.
-An assignment operator (=) saves the resulting CSV in the $J variable.
+The first command uses the `Start-Job` cmdlet to start a background job that runs a `Get-Process` command on the local computer.
+A pipeline operator (|) sends the resulting job object to the `ConvertTo-Csv` cmdlet, which converts the job object to CSV format.
```

Comment:
We don't need to talk about operators here. Simplify the text like I did in Example 1.